### DISCUSSION TITLE
[A-L] (2023/24) Foglio 2 - Esercizio 8

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#150] [Esercizi - Bertini] [answered]
![Es 8](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/1005591e-2955-4346-b654-273f707d4791)

**Comment by Elia-Belli** [comment]
![Senza titolo](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/27a2e44a-1fda-4551-97f6-5a5d8ded3af4)

**Comment by CiottoloMaggico** [comment, marked as answer]
**Soluzione del professore**
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/5137054c-2cac-4a74-bee0-dd37c8560a08)
